graphql-cli/src/main/java/io/ballerina/graphql/generator/ballerina/ClientGenerator.java

```diff
@@ -201,7 +201,7 @@ private ClassDefinitionNode generateClientClass(List<String> queryDocuments,
 
         return createClassDefinitionNode(metadataNode, createToken(PUBLIC_KEYWORD), classTypeQualifiers,
                 createToken(CLASS_KEYWORD), className, createToken(OPEN_BRACE_TOKEN),
-                createNodeList(members), createToken(CLOSE_BRACE_TOKEN));
+                createNodeList(members), createToken(CLOSE_BRACE_TOKEN), createToken(SEMICOLON_TOKEN));
```

Comment:
We can pass null here. @ushirask already sent a PR (https://github.com/ballerina-platform/graphql-tools/pull/95) for this last week, but the build failed due to the lang version IINM.